presto-base-jdbc/src/test/java/com/facebook/presto/plugin/jdbc/TestingDatabase.java

```diff
@@ -60,6 +60,7 @@ public TestingDatabase()
                 "('eleven', 'eleven', 11)," +
                 "('twelve', 'twelve', 12)" +
                 "");
+        connection.createStatement().execute("CREATE TABLE example.numbers_clone(text varchar primary key, text_short varchar(32), value bigint)");
```

Comment:
`text` is a confusing column name since this is a type name in many databases (including MySQL and PostgreSQL)